doc/pages/user/062-using_trigger_delivery_policies.md

```diff
@@ -0,0 +1,140 @@
+# Using Trigger Delivery Policies
+
+*Note: Trigger Delivery Policies are an experimental feature, see [Known Issues](#known-issues) for more information about their current status.*
+
+Trigger Delivery Policies allow customizing what Astarte is supposed to do in case of delivery errors on HTTP events
+and how to handle events that have not been delivered. More details on Trigger Delivery Policies can be found in the [Architecture Documentation](062-trigger_delivery_policies.html).
+
+Astarte allows you to install and delete Trigger Delivery Policies dynamically through its clients.
+A Trigger Delivery Policy is linked to an [HTTP Trigger](060-using_triggers.html) by specifying its name in the Trigger definition.
+A Trigger can be linked to at most one Trigger Delivery Policy, but a single Trigger Delivery Policy may serve any number of Triggers.
+
+## Listing Trigger Delivery Policies
+
+At any time, you can list existing Trigger Delivery Policies in a Realm and fetch their details and definitions.
+
+### Listing and querying Trigger Delivery Policies using Realm Management API
+
+To list all existing Trigger Delivery Policies in a Realm:
+
+`GET api.<your astarte domain>/realmmanagement/v1/<realm name>/policies`
+
+```json
+{
+  "data": [
+    "my_trigger_delivery_policy",
+    "simple_trigger_delivery_policy",
+    "other_trigger_delivery_policy",
+    "retry_upon_all_errors_delivery_policy"
+  ]
+}
+```
+
+To get a Trigger definition:
```

Comment:
To get a ~~Trigger~~**Policy** definition

Reply:
To get a ~~Policy~~ **Trigger Delivery Policy** definition